packages/manager/apps/web/client/app/hosting/cdn/shared/cors/hosting-cdn-shared-settings-cors.html

```diff
@@ -0,0 +1,55 @@
+<oui-modal
+    heading="Cross-Origin Resource Sharing"
```

Comment:
Not translated ?

Reply:
No CORS acronym should not be translated (same for all CDN settings)